consensus/spos/bls/subroundEndRound.go

```diff
@@ -17,7 +17,9 @@ type subroundEndRound struct {
 
 // SetAppStatusHandler method set appStatusHandler
 func (sr *subroundEndRound) SetAppStatusHandler(handler core.AppStatusHandler) {
```

Comment:
The pattern already used in app status handlers impl is that this func should return error if the provided handler is nil. Let's keep this pattern.

Reply:
SetAppStatusHandler return error now